pkgs/development/compilers/vyper/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv, lib, python3, fetchPypi, git, python3Packages }:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "vyper";
+  version = "0.2.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "06qi1rmilrajf90lj1cn811j1dx4a6pdbri4m16cz2874r1wy9fq";
+  };
+
+  nativeBuildInputs = [ git ];
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace 'asttokens==2.0.3' 'asttokens'
+  '';
+
+  propagatedBuildInputs = with python3Packages; [
+    asttokens
+    pycryptodome
+    pytest_xdist
+    pytestcov
+    recommonmark
+    semantic-version
+    sphinx
+    sphinx_rtd_theme
+  ];
+
+  disabledTests = [
```

Comment:
Do you mind saying why these are disabled

Reply:
Commenting out any one of those ignored tests results in the test failing, for instance if I comment out `"test_calldatacopy"`, the test fails with

```
[gw1] darwin -- Python 3.8.6 /nix/store/0mabx9absir73mmijrsv0dsyvyya463r-python3-3.8.6/bin/python3.8
file /private/tmp/nix-build-python3.8-vyper-0.2.7.drv-0/vyper-0.2.7/tests/compiler/test_calldatacopy.py, line 1
  def test_calldatacopy(get_contract_from_lll):
E       fixture 'get_contract_from_lll' not found
>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, cov, doctest_namespace, monkeypatch, no_cover, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, testrun_uid, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, worker_id
>       use 'pytest --fixtures [testpath]' for help on them.

/private/tmp/nix-build-python3.8-vyper-0.2.7.drv-0/vyper-0.2.7/tests/compiler/test_calldatacopy.py:
```

What could be causing this issue?  I'm not very familiar with pytest.